docs/troubleshooting.md

```diff
@@ -7,3 +7,55 @@ If you work on linux and cannot edit some of the project files right after the f
 ## TLS/HTTPS Issues
 
 See more in the [TLS section](tls.md)
+
+## Production issues
+
+### How to properly build fresh images for production use
+
+Remember that, by default, if you run `docker compose up -d`, only the files `compose.yaml` and `compose.override.yaml` will be used.
+See https://docs.docker.com/compose/intro/compose-application-model.
```

Comment:
What do you think, @tcoch, would it make sense to also add a link to information about how Docker Compose config file merge works?
```suggestion
See https://docs.docker.com/compose/intro/compose-application-model and https://docs.docker.com/compose/how-tos/multiple-compose-files/merge.
```